{"uid":"8aa8d4a8fa67b9d0","name":"test_network_cut","fullName":"tests.integration.ha_tests.test_self_healing#test_network_cut","historyId":"6b9f6734f6f55bf616f6dd52eb37e3bd","time":{"start":1707965590175,"stop":1707965695277,"duration":105102},"description":"Completely cut and restore network.","descriptionHtml":"<p>Completely cut and restore network.</p>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/3","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 448, in test_network_cut\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2939, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2845, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/3","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1707960481929,"stop":1707960481930,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1707960481930,"stop":1707960481930,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1707960481930,"stop":1707960482366,"duration":436},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1707963057461,"stop":1707963061765,"duration":4304},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1707965590174,"stop":1707965590174,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1707965590174,"stop":1707965590175,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Completely cut and restore network.","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/3","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 448, in test_network_cut\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2939, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/github-runner/_work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2845, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/3","steps":[],"attachments":[{"uid":"547820742f3e8a94","name":"log","source":"547820742f3e8a94.txt","type":"text/plain","size":1537}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1707965695279,"stop":1707965697379,"duration":2100},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1707965697380,"stop":1707965697380,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout::finalizer","time":{"start":1707966445727,"stop":1707966455785,"duration":10058},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1707966455788,"stop":1707966456372,"duration":584},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1707966456374,"stop":1707966456375,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1707966456375,"stop":1707966456375,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1707966456375,"stop":1707966456376,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-302-2a37937563820f7539d41bbf"},{"name":"thread","value":"9278-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":9,"skipped":0,"passed":6,"unknown":0,"total":17},"items":[{"uid":"f18e753a5ca1aea0","reportUrl":"../2000//#testresult/f18e753a5ca1aea0","status":"passed","time":{"start":1707790768297,"stop":1707791360559,"duration":592262}},{"uid":"a972f72311f018b3","reportUrl":"../1996//#testresult/a972f72311f018b3","status":"passed","time":{"start":1707705785172,"stop":1707706396935,"duration":611763}},{"uid":"f4a573492524fe53","reportUrl":"../1995//#testresult/f4a573492524fe53","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/3","time":{"start":1707618983602,"stop":1707619094223,"duration":110621}},{"uid":"9dc4b6395b80c8d3","reportUrl":"../1990//#testresult/9dc4b6395b80c8d3","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1707445672457,"stop":1707445756899,"duration":84442}},{"uid":"1f920ca8061a25c0","reportUrl":"../1988//#testresult/1f920ca8061a25c0","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7fe50ec9af20 state=finished raised AssertionError>]","time":{"start":1707361089062,"stop":1707361272442,"duration":183380}},{"uid":"a6acd91978c90137","reportUrl":"../1973//#testresult/a6acd91978c90137","status":"passed","time":{"start":1707272017573,"stop":1707272807862,"duration":790289}},{"uid":"117e4c5e265aba3f","reportUrl":"../1961//#testresult/117e4c5e265aba3f","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1707185589023,"stop":1707185672557,"duration":83534}},{"uid":"493023f57f26ead0","reportUrl":"../1957//#testresult/493023f57f26ead0","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/3","time":{"start":1707099871923,"stop":1707099955629,"duration":83706}},{"uid":"4d16dd2ace002da8","reportUrl":"../1956//#testresult/4d16dd2ace002da8","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1707012987935,"stop":1707013291853,"duration":303918}},{"uid":"a985de1c68e11eeb","reportUrl":"../1955//#testresult/a985de1c68e11eeb","status":"passed","time":{"start":1706927727367,"stop":1706928096065,"duration":368698}},{"uid":"678b484fd4c204bf","reportUrl":"../1939//#testresult/678b484fd4c204bf","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1706754533880,"stop":1706754604551,"duration":70671}},{"uid":"626e317862f47036","reportUrl":"../1935//#testresult/626e317862f47036","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1706673670270,"stop":1706673974428,"duration":304158}},{"uid":"fab3ab87f6991e7e","reportUrl":"../1928//#testresult/fab3ab87f6991e7e","status":"passed","time":{"start":1706580979053,"stop":1706581726462,"duration":747409}},{"uid":"833aeed512f429b2","reportUrl":"../1919//#testresult/833aeed512f429b2","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/2","time":{"start":1706494856758,"stop":1706495149969,"duration":293211}},{"uid":"1c7e47a38eecd054","reportUrl":"../1917//#testresult/1c7e47a38eecd054","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/3","time":{"start":1706408789641,"stop":1706409107420,"duration":317779}},{"uid":"424750c10a7fa06d","reportUrl":"../1916//#testresult/424750c10a7fa06d","status":"passed","time":{"start":1706321279312,"stop":1706321641318,"duration":362006}}]},"tags":["asyncio","group(1)"]},"source":"8aa8d4a8fa67b9d0.json","parameterValues":[]}